includes/class-wc-payments-account.php

```diff
@@ -1329,18 +1329,10 @@ private function init_stripe_onboarding( $wcpay_connect_from, $additional_args =
 			}
 			$account_data = [
 				'setup_mode'    => 'test',
-				'country'       => 'US',
 				'business_type' => 'individual',
 				'individual'    => [
 					'first_name' => 'John',
 					'last_name'  => 'Woolliams',
-					'address'    => [
-						'country'     => 'US',
-						'state'       => 'California',
-						'city'        => 'South San Francisco',
-						'line1'       => '1040 Grand Ave',
-						'postal_code' => '94080',
-					],
 					'ssn_last_4' => '0000',
```

Comment:
maybe comment / remove these not working prefills not to confuse developers? WDYT?

Reply:
Good point! Updated in 3c575bbf6.